test/contrib/plugs/plug_dependencies_test.py

```diff
@@ -0,0 +1,77 @@
+"""Tests for contrib.plugs.plug_dependencies."""
```

Comment:
I'm torn on whether to have tests for contrib stuff live in the tests directory (one of several problems I have with contrib's existence overall). The contrib.* import path is only valid if you're sitting in the top of the repo source, which granted you probably are if you're running unit tests, but it's misleading since contrib itself isn't actually a package.

Reply:
I'm not going to commit this CL as is.  The refactor will be implemented directly into the Plug Manager with tests in the appropriate location.